ruby.rb

```diff
@@ -0,0 +1,21 @@
+class Yo
+  def hello(name)
+    "hello there #{name}!"
+  end
+
+  def faillo(name)
+    'hello there! ' + name.to_s
+  end
+
+  def mappin
+    [1,2,3].map do |i|
```

Comment:
Space missing after comma.<br>Pass `&:to_s` as an argument to `map` instead of a block.